application/controllers/admin/CheckIntegrity.php

```diff
@@ -619,7 +620,10 @@ protected function checkintegrity()
         }
 
         /** Check for active surveys if questions are in the correct group **/
-        foreach ($oSurveys as $oSurvey) {
+        // Commented out the following code parts, because it is unlikely that the columns are not in the correct group
```

Comment:
We have it with some version, wen moving question : we don't move  subquestion if i remind.

But : we need a "ThisSurvey fixer" only for all of this.

Reply:
I think that I should uncomment the code and condition it so it only reachable by an URL parameter, like 

?fixResponseFields=y

Reply:
About integrity check : with ùmore then 2000 survey : it always broke … i disable the foreach …